src/GlobalHelper.php

```diff
@@ -175,7 +175,10 @@ public static function getCache(): CacheItemPoolInterface&CacheInterface
     public static function setupDefaultCache(): void
     {
         if (!isset(self::$cache)) {
-            self::setCache(new FilesystemAdapter(directory: sys_get_temp_dir() . '/castor'));
+            $home = getenv('HOME');
```

Comment:
HOME is unix specific. In the [PR about imports](https://github.com/jolicode/castor/pull/136/files#diff-6fc0821d7fdc2d529f8f13048d21bddca80da72c36bb666709aad791f63319acR30-R50), I did something more cross-platform inspired from what Composer itself does.